src/BuildingRegistry.Api.BackOffice.Handlers.Lambda/Requests/Building/ChangeBuildingOutlineLambdaRequest.cs

```diff
@@ -0,0 +1,59 @@
+namespace BuildingRegistry.Api.BackOffice.Handlers.Lambda.Requests.Building
+{
+    using Abstractions;
+    using Abstractions.Building;
+    using Be.Vlaanderen.Basisregisters.GrAr.Provenance;
+    using BuildingRegistry.Api.BackOffice.Abstractions.Building.Requests;
+    using BuildingRegistry.Building;
+    using BuildingRegistry.Building.Commands;
+    using Sqs.Requests.Building;
+
+    public sealed record ChangeBuildingOutlineLambdaRequest :
+        BuildingLambdaRequest,
+        IHasBackOfficeRequest<ChangeBuildingOutlineBackOfficeRequest>,
+        Abstractions.IHasBuildingPersistentLocalId
+    {
+        public ChangeBuildingOutlineBackOfficeRequest Request { get; }
+
+        public int BuildingPersistentLocalId { get; }
+
+        public ChangeBuildingOutlineLambdaRequest(
+            string messageGroupId,
+            ChangeBuildingOutlineSqsRequest sqsRequest)
+            : this(
+                messageGroupId,
+                sqsRequest.BuildingPersistentLocalId,
+                sqsRequest.TicketId,
+                sqsRequest.IfMatchHeaderValue,
+                sqsRequest.ProvenanceData.ToProvenance(),
+                sqsRequest.Metadata,
+                sqsRequest.Request)
+        { }
+
+        public ChangeBuildingOutlineLambdaRequest(
+            string messageGroupId,
+            int buildingPersistentLocalId,
```

Comment:
Maybe passing through the buildingPersistentLocalId is redundant as the messageGroupId should be equal to it?